docs/source/tutorial/collaboration-users.md

```diff
@@ -6,6 +6,10 @@
 It is recommended to use at least JupyterLab 3.6 with JupyterHub >= 3.1.1 for this.
 :::
 
+:::{note}
+If you use the JupyterLab >=4.0 you need to install the [jupyter-collaboration](https://github.com/jupyterlab/jupyter-collaboration) package in your single-user notebook instance.
```

Comment:
Let's be precise, since this has caused confusion over on JupyterLab itself: installing the jupyter-collaboration package _enables_ RTC, it's not just a requirement. There isn't any additional configuration or flags involved, and the standard way to disable it is to remove the package.

Reply:
Thanks for the reply.

If I try to use the `--collaborative` flag with the JupyterLab 4.0.3 I get the following error on startup:
```shell
(lab4test) C:\Users\lunin>jupyter lab --collaborative
[I 2023-08-01 14:06:24.904 ServerApp] Package jupyterlab took 0.0000s to import
[I 2023-08-01 14:06:24.935 ServerApp] Package jupyter_lsp took 0.0308s to import
[W 2023-08-01 14:06:24.935 ServerApp] A `_jupyter_server_extension_points` function was not found in jupyter_lsp. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[I 2023-08-01 14:06:24.951 ServerApp] Package jupyter_server_terminals took 0.0181s to import
[I 2023-08-01 14:06:24.951 ServerApp] Package notebook_shim took 0.0000s to import
[W 2023-08-01 14:06:24.951 ServerApp] A `_jupyter_server_extension_points` function was not found in notebook_shim. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[I 2023-08-01 14:06:24.966 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2023-08-01 14:06:24.966 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2023-08-01 14:06:24.982 ServerApp] jupyterlab | extension was successfully linked.
[I 2023-08-01 14:06:25.435 ServerApp] notebook_shim | extension was successfully linked.
[I 2023-08-01 14:06:25.602 ServerApp] notebook_shim | extension was successfully loaded.
[I 2023-08-01 14:06:25.617 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2023-08-01 14:06:25.617 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2023-08-01 14:06:25.617 LabApp] JupyterLab extension loaded from C:\Users\lunin\AppData\Local\mambaforge\envs\lab4test\Lib\site-packages\jupyterlab
[I 2023-08-01 14:06:25.617 LabApp] JupyterLab application directory is C:\Users\lunin\AppData\Local\mambaforge\envs\lab4test\share\jupyter\lab
[I 2023-08-01 14:06:25.617 LabApp] Extension Manager is 'pypi'.
[C 2023-08-01 14:06:25.617 LabApp]
    To enable real-time collaboration, you must install the extension `jupyter_collaboration`.
    You can install it using pip for example:

      python -m pip install jupyter_collaboration
```
while the same command works in JupyterLab 3.6.5:

```shell
(lab36test) C:\Users\lunin>jupyter lab --collaborative
[I 2023-08-01 14:14:56.137 ServerApp] Package jupyterlab took 0.0000s to import
[I 2023-08-01 14:14:56.137 ServerApp] Package jupyter_server_fileid took 0.0058s to import
[I 2023-08-01 14:14:56.168 ServerApp] Package jupyter_server_terminals took 0.0188s to import
[I 2023-08-01 14:14:56.231 ServerApp] Package jupyter_server_ydoc took 0.0639s to import
[I 2023-08-01 14:14:56.231 ServerApp] Package nbclassic took 0.0000s to import
[W 2023-08-01 14:14:56.231 ServerApp] A `_jupyter_server_extension_points` function was not found in nbclassic. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[I 2023-08-01 14:14:56.231 ServerApp] Package notebook_shim took 0.0000s to import
[W 2023-08-01 14:14:56.231 ServerApp] A `_jupyter_server_extension_points` function was not found in notebook_shim. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[I 2023-08-01 14:14:56.246 ServerApp] jupyter_server_fileid | extension was successfully linked.
[I 2023-08-01 14:14:56.262 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2023-08-01 14:14:56.277 ServerApp] jupyter_server_ydoc | extension was successfully linked.
[I 2023-08-01 14:14:56.293 ServerApp] jupyterlab | extension was successfully linked.
[I 2023-08-01 14:14:56.309 ServerApp] nbclassic | extension was successfully linked.
[I 2023-08-01 14:14:56.764 ServerApp] notebook_shim | extension was successfully linked.
[I 2023-08-01 14:14:57.077 ServerApp] notebook_shim | extension was successfully loaded.
[I 2023-08-01 14:14:57.077 FileIdExtension] Configured File ID manager: ArbitraryFileIdManager
[I 2023-08-01 14:14:57.092 FileIdExtension] ArbitraryFileIdManager : Configured root dir: C:/Users/lunin
[I 2023-08-01 14:14:57.092 FileIdExtension] ArbitraryFileIdManager : Configured database path: C:\Users\lunin\AppData\Roaming\jupyter\file_id_manager.db
[I 2023-08-01 14:14:57.092 FileIdExtension] ArbitraryFileIdManager : Successfully connected to database file.
[I 2023-08-01 14:14:57.092 FileIdExtension] ArbitraryFileIdManager : Creating File ID tables and indices with journal_mode = DELETE
[I 2023-08-01 14:14:57.108 FileIdExtension] Attached event listeners.
[I 2023-08-01 14:14:57.108 ServerApp] jupyter_server_fileid | extension was successfully loaded.
[I 2023-08-01 14:14:57.108 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2023-08-01 14:14:57.108 ServerApp] jupyter_server_ydoc | extension was successfully loaded.
[I 2023-08-01 14:14:57.123 LabApp] JupyterLab extension loaded from C:\Users\lunin\AppData\Local\mambaforge\envs\lab36test\Lib\site-packages\jupyterlab
[I 2023-08-01 14:14:57.123 LabApp] JupyterLab application directory is C:\Users\lunin\AppData\Local\mambaforge\envs\lab36test\share\jupyter\lab
[I 2023-08-01 14:14:57.123 ServerApp] jupyterlab | extension was successfully loaded.
```

Reply:
Sorry, this is what I meant:

```suggestion
Starting with JupyterLab >=4.0, installing the [jupyter-collaboration](https://github.com/jupyterlab/jupyter-collaboration) package in your single-user environment enables collaborative mode, instead of passing the `--collaborative` flag at runtime.
```

conveying that the installation command _is_ the enabling step, rather than a new _requirement_, with an unchanged enabling. Several folks have been confused that installing jupyter-collaboration without `--collaborative` still enables RTC.